### PR TITLE
bug(1868848): Skipping copying checks for baseline tables for apps marked as not receiving the baseline ping

### PR DESCRIPTION
**Pull request by kik-kik**
# bug(1868848): Skipping copying checks for baseline tables for apps marked as not receiving the baseline ping

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-2154)

### CODE REVIEW

**Review by fbertsch — APPROVED** [review]
I have a few nits with this, but we can merge this and fix them later:
1. I believe that `does not send baseline ping` should be indicated in a Glean app's `repositories.yaml` entry. This would be useful information elsewhere as well (e.g. the Glean dictionary would not index the baseline ping)
2. It would be nice to have a way to verify that these are not sending that data

File: sql_generators/glean_usage/common.py [hunk 1]
```diff
@@ -18,6 +18,24 @@
...
+    "burnham",
+    "firefox_reality_pc",
+    "lockwise_android",
+    "mach",
+    "monitor_cirrus",
+    "moso_mastodon_backend",
+    "mozphab",
+    "mozregression",
+)
+
```
Comment (scholtzan) [inline sql_generators/glean_usage/common.py:38]: I think we should move this into the config file here: https://github.com/mozilla/bigquery-etl/blob/6e2e8f6f688a91f292774995f119ecdd0b63cd8f/bqetl_project.yaml#L433

Comment (kik-kik) [reply sql_generators/glean_usage/common.py:38]: @scholtzan  That sounds like a reasonable suggestion, where would be the correct place to file a ticket for this so we do not forget about it?

Comment (scholtzan) [reply sql_generators/glean_usage/common.py:38]: There is a PR here: https://github.com/mozilla/bigquery-etl/pull/4676

**Pull request merged**